src/bpftrace.cpp

```diff
@@ -38,6 +38,7 @@ namespace bpftrace {
 
 DebugLevel bt_debug = DebugLevel::kNone;
 bool bt_verbose = false;
+volatile sig_atomic_t bt_sigint_called = false;
```

Comment:
*maybe* the finalize_ variable could be repurposed for this end instead of creating a new variable just for that.

Reply:
I don't think we can use `finalize_` here, because modifying it would require a closure over the bpftrace object (which is not allowed in function pointers). I'll try to use a static member instead.

Reply:
I meant to turn `finalize_` into a static member and then using it instead of `bt_sigint_called`...

Reply:
I don't think that's a good idea, that would mean that if you called request_finalize on one BPFTrace object, they would all exit, which dosen't seem very intuitive.